runtime/src/accounts_db.rs

```diff
@@ -981,6 +1100,17 @@ impl AccountsDB {
                                     );
                                 }
 
+                                // prune zero_lamport_pubkey set which should contain all 0-lamport
+                                // keys whether rooted or not. A 0-lamport update may become rooted
+                                // in the future.
+                                let has_zero_lamport_accounts = locked_entry
+                                    .slot_list()
```

Comment:
nits: also I'd like to have the `let slot_list = ...` nicety for our prosperity. :)